tools/flang2/flang2exe/lldebug.cpp

```diff
@@ -1873,7 +1899,7 @@ lldbg_emit_parameter_list(LL_DebugInfo *db, DTYPE dtype, DTYPE ret_dtype,
 
   /* do the return value, if it appears in the argument list */
   num_args = 0;
-  if (fval && SCG(fval) == SC_DUMMY) {
+  if (!ret_dtype && fval && SCG(fval) == SC_DUMMY) {
```

Comment:
This is not an obvious change. Could you clarify in the the comment that `ret_dtype ` will be `DT_NONE` when `sptr` refers to a procedure pointer or when the return value is a pointer itself, and also explain why we don't need the debuginfo metadata for the return value of a normal non-pointer procedure?

Reply:
@alokkrsharma can you address this comment?

Reply:
This change is not required, I have removed it. @bryanpkc can you please have a look again ?

Reply:
OK thanks.